server/test/views/admin/question/QuestionConfigTest.java

```diff
@@ -35,7 +43,8 @@ public void resultForAllQuestions(QuestionType questionType) throws Exception {
     }
 
     QuestionForm questionForm = QuestionFormBuilder.create(questionType);
-    Optional<DivTag> maybeConfig = QuestionConfig.buildQuestionConfig(questionForm, messages);
+    Optional<DivTag> maybeConfig =
+        QuestionConfig.buildQuestionConfig(questionForm, messages, false);
```

Comment:
Kinda related to the above - shouldn't these be set to true, so that removing the feature flag is a noop? Or if not now then when?